setup.py

```diff
@@ -41,5 +41,7 @@
     keywords="computer-vision",
     packages=find_packages(),
     python_requires=">= 3.6",
-    install_requires=[],
+    install_requires=[
```

Comment:
can we move it to the conda environment file?

Reply:
@ayushbaid  Only hydra >=1.10 supports recursive object instantiation, and I didnt see 1.1.0 in conda (only available via pip install --pre)

Reply:
Can we still move it to the yml file? (like we did for degensac)

Reply:
sure, forgot that was possible